examples/basic_example_v2/basic_example.py

```diff
@@ -0,0 +1,332 @@
+"""
+
+
+def fill_tables_with_data(pool, path):
+    print("\nFilling tables with data...")
+
+    global FillDataQuery
+
+    def callee(session):
+
+        prepared_query = FillDataQuery.format(path)
+        with session.transaction(ydb.QuerySerializableReadWrite()).execute(
+            prepared_query,
+            {
+                "$seriesData": (basic_example_data.get_series_data(), basic_example_data.get_series_data_type()),
+                "$seasonsData": (basic_example_data.get_seasons_data(), basic_example_data.get_seasons_data_type()),
+                "$episodesData": (basic_example_data.get_episodes_data(), basic_example_data.get_episodes_data_type()),
+            },
+            commit_tx=True,
+        ) as _:
+            pass
+
+    return pool.retry_operation_sync(callee)
+
+
+def select_simple(pool, path):
+    print("\nCheck series table...")
+
+    def callee(session):
+        # new transaction in serializable read write mode
+        # if query successfully completed you will get result sets.
+        # otherwise exception will be raised
+        with session.transaction(ydb.QuerySerializableReadWrite()).execute(
+            """
+            PRAGMA TablePathPrefix("{}");
+            $format = DateTime::Format("%Y-%m-%d");
+            SELECT
+                series_id,
+                title,
+                $format(DateTime::FromSeconds(CAST(DateTime::ToSeconds(DateTime::IntervalFromDays(CAST(release_date AS Int16))) AS Uint32))) AS release_date
+            FROM series
+            WHERE series_id = 1;
+            """.format(
+                path
+            ),
+            commit_tx=True,
+        ) as result_sets:
+            first_set = next(result_sets)
+            for row in first_set.rows:
+                print(
+                    "series, id: ",
+                    row.series_id,
+                    ", title: ",
+                    row.title,
+                    ", release date: ",
+                    row.release_date,
+                )
+
+            return first_set
+
+    return pool.retry_operation_sync(callee)
+
+
+def upsert_simple(pool, path):
+    print("\nPerforming UPSERT into episodes...")
+
+    def callee(session):
+        with session.transaction().execute(
+            """
+            PRAGMA TablePathPrefix("{}");
+            UPSERT INTO episodes (series_id, season_id, episode_id, title) VALUES (2, 6, 1, "TBD");
+            """.format(
+                path
+            ),
+            commit_tx=True,
+        ) as _:
+            pass
+
+    return pool.retry_operation_sync(callee)
+
+
+def select_with_parameters(pool, path, series_id, season_id, episode_id):
+    def callee(session):
+        query = """
+        PRAGMA TablePathPrefix("{}");
+        $format = DateTime::Format("%Y-%m-%d");
+        SELECT
+            title,
+            $format(DateTime::FromSeconds(CAST(DateTime::ToSeconds(DateTime::IntervalFromDays(CAST(air_date AS Int16))) AS Uint32))) AS air_date
```

Comment:
It seems very difficult, do you really need the construction with many casts and call functions?